src/Classes/PartyTab.lua

```diff
@@ -0,0 +1,620 @@
+	child = { elem = "ExportedBuffs", attrib = { name = "EnemyConditions" } }
+	child.attrib.string = self:exportBuffs("EnemyConditions")
+	t_insert(xml, child)
+	child = { elem = "ExportedBuffs", attrib = { name = "EnemyMods" } }
+	child.attrib.string = self:exportBuffs("EnemyMods")
+	t_insert(xml, child)
+	self.lastContentAura = self.controls.editAuras.buf
+	self.lastContentCurse = self.controls.editCurses.buf
+	self.lastContentEnemyCond = self.controls.enemyCond.buf
+	self.lastContentEnemyMods = self.controls.enemyMods.buf
+	self.lastEnableExportBuffs = self.enableExportBuffs
+end
+
+function PartyTabClass:Draw(viewPort, inputEvents)
+	self.x = viewPort.x
+	self.y = viewPort.y
+	self.width = viewPort.width
+	self.height = viewPort.height
+
+	for id, event in ipairs(inputEvents) do
+		if event.type == "KeyDown" then
+			if event.key == "z" and IsKeyDown("CTRL") then
+				if self.controls.editAuras.hasFocus then
+					self.controls.editAuras:Undo()
+				elseif self.controls.editCurses.hasFocus then
+					self.controls.editCurses:Undo()
+				end
+			elseif event.key == "y" and IsKeyDown("CTRL") then
+				if self.controls.editAuras.hasFocus then
+					self.controls.editAuras:Redo()
+				elseif self.controls.editCurses.hasFocus then
+					self.controls.editCurses:Redo()
+				end
+			end
+		end
+	end
+	self:ProcessControlsInput(inputEvents, viewPort)
+
+	main:DrawBackground(viewPort)
+
+	self:DrawControls(viewPort)
+
+	self.modFlag = (self.lastContentAura ~= self.controls.editAuras.buf 
+			or self.lastContentCurse ~= self.controls.editCurses.buf
+			or self.lastContentEnemyCond ~= self.controls.enemyCond.buf
+			or self.lastContentEnemyMods ~= self.controls.enemyMods.buf
+			or self.lastEnableExportBuffs ~= self.enableExportBuffs)
+end
+
+function PartyTabClass:ParseTags(line, currentModType) -- should parse this correctly instead of string match
+	if not line then
+		return "none", {}
+	end
+	local extraTags = {}
+	local modType = currentModType
+	for line2 in line:gmatch("([^,]*),?") do
+		if line2:find("type=GlobalEffect/") then
+			if line2:find("type=GlobalEffect/effectType=AuraDebuff") then
+				t_insert(extraTags, {
+					type = "GlobalEffect",
+					effectType = "AuraDebuff"
+				})
+				modType = "AuraDebuff"
+			elseif line2:find("type=GlobalEffect/effectType=Aura") then
+				t_insert(extraTags, {
+					type = "GlobalEffect",
+					effectType = "Aura"
+				})
+				modType = "Aura"
+			elseif line2:find("type=GlobalEffect/effectType=Curse") then
+				t_insert(extraTags, {
+					type = "GlobalEffect",
+					effectType = "Curse"
+				})
+				modType = "Curse"
+			end
+		elseif line2:find("type=Condition/") then
+			if line2:find("type=Condition/neg=true/var=RareOrUnique") then
+				t_insert(extraTags, {
+					type = "Condition",
+					neg = true,
+					var = "RareOrUnique"
+				})
+			elseif line2:find("type=Condition/var=RareOrUnique") then
+				t_insert(extraTags, {
+					type = "Condition",
+					var = "RareOrUnique"
+				})
+			end
+		end
```

Comment:
This feels pretty brittle and hardcoded.  You should be able to split on `|`, `/`, and `=` to get all of the things you need.

Ninja edit: ah, I see the comment at the top of this function saying as such

Reply:
yeah this needs improvement but I held back due to needing a better method for enemy conditions/mods, becouse those done exist I will make this more generalized for these and update them when I add back conditions etc